src/content/plugins/module-concatenation-plugin.md

```diff
@@ -16,3 +16,78 @@ new webpack.optimize.ModuleConcatenationPlugin()
 ```
 
 > Scope Hoisting is specifically a feature made possible by ECMAScript Module syntax. Because of this webpack may fallback to normal bundling based on what kind of modules you are using, and [other conditions](https://medium.com/webpack/webpack-freelancing-log-book-week-5-7-4764be3266f5).
+
+# Optimization Bailouts
```

Comment:
Can you change this to an `h2` (i.e. `##`)? The beginning of every page starts with an `h1` with the page title and, aside from keeping the design consistent, it's not good to switch back and forth for SEO reasons. Also we typically drop two new lines between `h2` sections.

Reply:
Sure!